magnetic/Generic/gfxlink.c

```diff
@@ -64,9 +64,29 @@
 #define DIRSEP "/"
 #endif
 
+#include <limits.h>
```

Comment:
This file is a separate utility, not part of the interpreter itself, so it probably doesn't need any updates at all.

If you _do_ want to keep it updated, it should get the same `stdint.h` treatment as in `defs.h`, instead of using limits to determine sizes.

But really, I recommend reverting this and keeping it as-is (or even removing this file).